webapp/jaasai/views.py

```diff
@@ -146,7 +146,12 @@ def blog_feed():
 
 
 @jaasai.route("/robots.txt")
-def static_from_root():
+def robots_from_root():
+    return send_from_directory(current_app.static_folder, request.path[1:])
+
+
+@jaasai.route("/favicon.ico")
+def favicon_from_root():
```

Comment:
As these functions are the same I think you could direct both routes to the same function. For example:
```python
@jaasai.route("/robots.txt")
@jaasai.route("/favicon.ico")
def assets_from_root():
    return send_from_directory(current_app.static_folder, request.path[1:])
```